commit de04b4e0fa5c6908e3a3d78028e6066a176f80f0
Author: David Teigland <teigland@redhat.com>
Date:   Mon Nov 5 14:53:23 2012 -0600

    stonith-helper: use reboot instead of off
    
    That 0 means reboot was verified by
    Jacek Konieczny <jajcus@jajcus.net>
    
    Signed-off-by: David Teigland <teigland@redhat.com>

diff --git a/fence/stonith_helper.c b/fence/stonith_helper.c
index 5b384c1..32d2c0d 100644
--- a/fence/stonith_helper.c
+++ b/fence/stonith_helper.c
@@ -77,7 +77,7 @@ int main(int argc, char *argv[])
 	if (t >= fail_time)
 		return 0;
 
-	rv = stonith_api_kick_helper(nodeid, 300, 1);
+	rv = stonith_api_kick_helper(nodeid, 300, 0);
 	if (rv) {
 		fprintf(stderr, "kick_helper error %d nodeid %d\n", rv, nodeid);
 		openlog("stonith_helper", LOG_CONS | LOG_PID, LOG_DAEMON);
